.babelrc

```diff
@@ -3,16 +3,17 @@
     ["@babel/env", {
       "targets": {
         "browsers": ["last 2 versions"]
-      }
+      },
+      "useBuiltIns": "usage"
     }],
     "@babel/react",
     "@babel/preset-typescript"
   ],
   "plugins": [
     ["@babel/plugin-proposal-decorators", { "legacy": true }],
     "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-transform-runtime",
     "inline-react-svg",
+    "lodash",
```

Comment:
I think it's safe to remove lodash here

Reply:
Even though `underscore` is the official utility lib it's being used all over (same in force).

Reply:
I wonder if there's some kind of codemod we could run to swap it out

Reply:
You mean remove underscore? I'd been encouraged to switch to `lodash` wherever possible.

Reply:
Yup remaining underscore usage and replace with lodash using the codemod, would automate all the work away.

Reply:
Thinking further though, that codemod won't work with coffeescript /shrug